TestTools/StreamChatTestMockServer/StreamChatTestMockServer.swift

```diff
@@ -34,6 +33,6 @@ extension Bundle {
     }()
 
     var pathToJSONsFolder: String {
-        Self.testTools.bundlePath.contains(Self.bundleName) ? Self.resourcesJSONs : ""
+        ""
```

Comment:
Is this correct?

Reply:
Yeah, this is the workaround to fix the warning in SwiftUI repo. It might not look great, but it follows the pattern of having  `pathToJSONsFolder` in both `TestTools` and `MockServer`.